content/rs/concepts/memory-architecture/memory-management.md

```diff
@@ -1,40 +1,35 @@
 ---
-Title: Memory Mangement with Redis Enterprise Software (RES)
+Title: Memory Mangement with Redis Enterprise Software (RS)
 description: 
 weight: $weight
 alwaysopen: false
 ---
-By default, RES manages a node's memory so data is entirely in RAM for
-performance reasons. The exception to this is RES's [Redis on
-Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}})
-feature where Flash memory (SSDs) can be used to store data too. If
-there's not enough RAM available, RES does not allow inserting of more
-data into databases. This limitation is intentional and ultimately
-beneficial. It is detrimental to RES for the OS to take control of
-allocating memory and spill some data to the swap, thus managing memory
-on RES's behalf. RES has been designed to know better than the OS what
-it needs and when it needs it. RES protects the existing data and
-prevents the database from being able to store data into the shards. It
+By default, RS manages node memory so that data is entirely in RAM for improved
+database performance. RS is designed to handle memory management to optimize database 
+performance - better than OS memory management. If not enough RAM is available, 
+RS prevents adding more data into the databases. RS protects the existing data and
+prevents the database from being able to store data into the shards. It 
 can be configured to move the data to another node, or even discard it.
-This depends on [eviction
-policy]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}})
-set on a database by the administrator.
+The behavior depends on the [eviction policy]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}})
+set on each database by the administrator.
 
-With this memory management style, comes the responsibility to monitor
-the nodes, clusters, databases, etc., but you are already doing that,
-right?
+RoF [Redis on Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}})
+manages memory in a different way so that Flash memory (SSDs) can also be used 
+to store data. 
 
 ## What happens when Redis Enterprise Software is low on RAM?
```

Comment:
@ushachar Does the numbered mitigation flow accurately reflect the process?
IOW, is it ready to merge?

Reply:
The order seems correct (assuming memory growth is slow enough so that all actions won't be applied simultaneity). 
I suppose we can add a lot of details here making the description more accurate, but then we'll need to update it when nuances in our internal logic changes, so let's not.
p.s. i would change "(OOM) messages" to "(-OOM) responses"

Reply:
I'll call it order of priority.
Changed to OOM responses.